src/java/org/apache/cassandra/schema/IndexMetadata.java

```diff
@@ -67,7 +67,8 @@ public final class IndexMetadata
 
     static
     {
-        indexNameAliases.put(StorageAttachedIndex.class.getSimpleName(), StorageAttachedIndex.class.getCanonicalName());
+        indexNameAliases.put(StorageAttachedIndex.NAME, StorageAttachedIndex.class.getCanonicalName());
```

Comment:
should we change keyword from "sai" to "StorageAttachedIndex" ? as I think StorageAttachedIndex is more expressive than sai to express the situation of indexing clearly. 

Sorry, I may be missing some final discussion process for the content of DISCUSS, so I will finally express the idea of ​​​​replacing the sai keyword.

Reply:
You can also use "StorageAttachedIndex" (see the line below).